sorts/recursive_bubble_sort.py

```diff
@@ -1,41 +1,41 @@
-def bubble_sort(list1):
+def bubble_sort(list_data: list, length: int) -> list:

```

Comment:
```suggestion
def bubble_sort(list_data: list, length: int=0) -> list:
    length = length or len(list_data)
```

Reply:
Why need change like this ?

Reply:
If I want to sort a list, why do I need to tell you how long it is in a language like Python that has len() builtin?

Reply:
Did you mean optional argument ? can you edit like this?
``` python
def bubble_sort(list_data: list, length: int = 0) -> list:
```